code/scripting/api/objs/object.cpp

```diff
@@ -171,6 +172,25 @@ ADE_VIRTVAR(LastOrientation, l_Object, "orientation", "Object world orientation
 	return ade_set_args(L, "o", l_Matrix.Set(matrix_h(&objh->objp->last_orient)));
 }
 
+ADE_VIRTVAR(ModelInstance, l_Object, "modelinstance", "model instance used by this object", "modelinstance", "Model instance, nil if this object does not have one, or invalid model instance handle if object handle is invalid")
```

Comment:
the second "modelinstance" string should be nullptr if the value cannot be assigned. This will correctly display non-assignability in the documentation

Reply:
Ok